go/api/generic_commands.go

```diff
@@ -30,4 +30,30 @@ type GenericBaseCommands interface {
 	//
 	// [valkey.io]: https://valkey.io/commands/del/
 	Del(keys []string) (Result[int64], error)
+
+	// Copy copies the value stored at the source key to the destination key.
+	//
+	// By default, the destination key is created in the same logical database used by the connection. However, the `DB` option allows specifying an alternative logical database index for the destination key. If the destination key already exists, the command returns zero. The `REPLACE` option can be used to remove the destination key before copying the value.
+	//
+	// Note:
+	// - When in cluster mode, the command may route to multiple nodes depending on the location of the source and destination keys.
+	//
+	// Parameters:
+	//  sourceKey      - The key from which to copy the value.
+	//  destinationKey - The key to copy the value to.
+	//  destinationDB  - An optional integer representing the database index for the destination key. If nil, the current database is used.
```

Comment:
How we do in docs - nil or `nil`?
```suggestion
	//  destinationDB  - An optional integer representing the database index for the destination key. If `nil`, the current database is used.
```